src/css/profile/mobile/common/masteronoffswitch.less

```diff
@@ -21,12 +21,43 @@
         display: flex;
         align-items: center;
 
+        &::before {
+            content: "";
+            width: 100%;
+            height: 64 * @px_base;
+                opacity: 0;
```

Comment:
Indent is not proper

Reply:
done

Reply:
You not commited changes for this file probably

Reply:
this time it should be ok